src/OpenTelemetry/Trace/Export/ExportComponentBase.cs

```diff
@@ -1,35 +0,0 @@
-﻿// <copyright file="ExportComponentBase.cs" company="OpenTelemetry Authors">
-// Copyright 2018, OpenTelemetry Authors
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-// </copyright>
-
-namespace OpenTelemetry.Trace.Export
-{
-    public abstract class ExportComponentBase : IExportComponent
```

Comment:
I guess abstract class is still needed to be a base class for NoopExportComponent , ExportComponent, and any external implementations. 

Then someone can use span stores without thinking about the particular implementation.

Reply:
Alright.  :)  Including the abstract base class doesn't remove the explicit casting anyway since IExportComponent doesn't include the span store methods.  To keep callers from having to think about implementation the two level interface was what came to mind previously.